Libraries/Image/resolveAssetSource.js

```diff
@@ -25,6 +25,7 @@ var AssetRegistry = require('AssetRegistry');
 var PixelRatio = require('PixelRatio');
 var Platform = require('Platform');
 var SourceCode = require('NativeModules').SourceCode;
+var assetPathUtils = require('../../local-cli/bundle/assetPathUtils');
```

Comment:
Yeah, that's one of the reasons I hate Github's review flow (Phabricator is so much better)